contracts/modules/common/RelayerModuleV2.sol

```diff
@@ -0,0 +1,343 @@
+// Copyright (C) 2018  Argent Labs Ltd. <https://argent.xyz>
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.5.4;
+import "../../wallet/BaseWallet.sol";
+import "../../interfaces/Module.sol";
+import "../../utils/GuardianUtils.sol";
+import "./BaseModule.sol";
+
+/**
+ * @title RelayerModuleV2
+ * @dev Base module containing logic to execute transactions signed by eth-less accounts and sent by a relayer.
+ * RelayerModuleV2 should ultimately replace RelayerModule and be subclassed by all modules.
+ * It is currently only subclassed by RecoveryManager and ApprovedTransfer.
+ * @author Julien Niset - <julien@argent.im>
```

Comment:
You should either remove my name or add yours :-)

Reply:
Done :)